backend/api/job.proto

```diff
@@ -61,18 +60,33 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 
 
 service JobService {
-  // Creates a new job.
-  rpc CreateJob(CreateJobRequest) returns (Job) {
+  // Creates a new job in an experiment, given the experiment ID.
+  rpc CreateJobByExperiment(CreateJobByExperimentRequest) returns (Job) {
     option (google.api.http) = {
-      post: "/apis/v1beta1/jobs"
+      post: "/apis/v2beta1/experiments/{id}/jobs"
```

Comment:
```suggestion
      post: "/apis/v2beta1/experiments/{experiment_id}/jobs"
```

Reply:
experiment_id field is defined under `CreateJobByExperimentRequest` but written as id here